.github/workflows/pr-title-lint.yml

```diff
@@ -0,0 +1,87 @@
+name: Lint Pull Request Title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions:
+  contents: read
+
+jobs:
+  lint_commit:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Commitlint
+        shell: bash
+        run: npm install --location=global @commitlint/cli@17.1.1
+
+      - name: Lint Commit
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -ex
+          PR_TITLE="${{ github.event.pull_request.title }}"
+
+          ERR='
+            The PR title `${{ github.event.pull_request.title }}` does not follow the Calimero-Network Commit Message Convention
+            https://github.com/calimero-network/core/blob/master/CONTRIBUTING.md#commit-message-style
+
+            Our PR title should have the following structure.
+
+              <type>(<scope>): <subject>
+              <BLANK LINE>
+              <body>
+              <BLANK LINE>
+              <footer>
```

Comment:
this is linting the PR title, we can't have a "body" in the title

Reply:
Updated